orchagent/portsorch.cpp

```diff
@@ -910,7 +919,14 @@ bool PortsOrch::createBindAclTableGroup(sai_object_id_t id, sai_object_id_t &gro
                 bind_type = SAI_ACL_BIND_POINT_TYPE_LAG;
                 break;
             case Port::VLAN:
-                bind_type = SAI_ACL_BIND_POINT_TYPE_VLAN;
+                if (port.m_rif_id)
+                {
+                    bind_type = SAI_ACL_BIND_POINT_TYPE_ROUTER_INTERFACE;
```

Comment:
Why is RIF accounted only for Vlan. I mean Port and Lag (cases above) can also have corresponding RIF.

Reply:
This PR is only meant to support VLAN RIFs.